java/java.editor.base/test/unit/src/org/netbeans/modules/java/editor/base/semantic/TestBase.java

```diff
@@ -35,6 +35,7 @@
 import java.util.Collection;
 import java.util.Comparator;
 import java.util.EnumSet;
+import java.util.Iterator;
```

Comment:
Ideally, unnecessary changes should be undone.

Reply:
Comment addressed.